Vizzuality/heco-invest
# Implement the static content translation strategy

We [previously identified the need](https://vizzuality.atlassian.net/browse/LET-79) to use Transifex Native instead of Transifex Live in order to translate the platform's static content. In this PR, this strategy has been put into practice:
-  Next.js is configured with the Spanish, English and Portuguese locales, to default to Spanish, and to automatically detect and redirect the user according to their browser's locale
- When the user changes the language in the language selector, they are redirected to the appropriate URL (`/` for Spanish, `/en` for English or `/pt` for Portuguese) and a cookie is set so that Next.js will remember their locale
- Transifex Native is set to use Next.js' locale, and all the current static content is passed though Transifex' `<T />` component or `t` function (coming from the hook `useT`)
- The reusable GitHub workflow is updated to push the new static strings to Transifex when deployed to staging or production
- React Aria is set to use Next.js' locale (React Aria provides its own translations in all 3 languages)

## Testing instructions

- Make sure that you're redirected to the URL corresponding to the language of your browser
- Make sure that if you change the language of the platform, it is remembered

## Tracking

[LET-80](https://vizzuality.atlassian.net/browse/LET-80).
